p2p/peer.go

```diff
@@ -484,11 +494,14 @@ func (p *peer) metricsReporter() {
 		case <-p.metricsTicker.C:
 			status := p.mconn.Status()
 			var sendQueueSize float64
+			queues := make(map[byte]int, len(status.Channels))
 			for _, chStatus := range status.Channels {
 				sendQueueSize += float64(chStatus.SendQueueSize)
+				queues[chStatus.ID] = chStatus.SendQueueSize
 			}
 
 			p.metrics.PeerPendingSendBytes.With("peer_id", string(p.ID())).Set(sendQueueSize)
+			schema.WritePendingBytes(p.traceClient, string(p.ID()), queues)
```

Comment:
We could trace the Channel ID as well.

Reply:
I think we do right? queues is a map between the channel ID and the SendQueueSize